### PR TITLE
Changing default localstack credentials

### PR DESCRIPTION
**Pull request by fokion**
the default credentials according to the localstack are test , when we do `getAccessKey` and` getSecretKey` we will need to check the environment map if we have the following keys 
 - AWS_ACCESS_KEY_ID
 -  AWS_SECRET_ACCESS_KEY 
and fallback to the default values defined in a static final variable with the value `test` based on this https://github.com/localstack/localstack/blob/master/doc/interaction/README.md?plain=1#L34

### CODE REVIEW

**Comment by fokion** [comment]
this is ready to be reviewed @eddumelendez

File: modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java [hunk 1]
```diff
@@ -451,4 +455,53 @@ private String runAwsCliAgainstDockerNetworkContainer(String command) throws Exc
             return logs;
         }
     }
+
+    public static class WithEnvironmentVariables {
+
+        @ClassRule
+        public static LocalStackContainer localstack = new LocalStackContainer(LocalstackTestImages.LOCALSTACK_LATEST)
+            .withServices(Service.S3)
```
Comment (eddumelendez) [inline modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java:463]:
```suggestion
```

File: modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackTestImages.java [hunk 2]
```diff
@@ -9,5 +9,7 @@ public interface LocalstackTestImages {
     DockerImageName LOCALSTACK_0_11_IMAGE = LOCALSTACK_IMAGE.withTag("0.11.3");
     DockerImageName LOCALSTACK_0_12_IMAGE = LOCALSTACK_IMAGE.withTag("0.12.8");
     DockerImageName LOCALSTACK_0_13_IMAGE = LOCALSTACK_IMAGE.withTag("0.13.0");
+
+    DockerImageName LOCALSTACK_LATEST = LOCALSTACK_IMAGE.withTag("latest");
```
Comment (eddumelendez) [inline modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackTestImages.java:13]: Using latest tags is not recommended. Use an specific tag, please

File: modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java [hunk 3]
```diff
@@ -451,4 +455,53 @@ private String runAwsCliAgainstDockerNetworkContainer(String command) throws Exc
...
+            .withEnv("S3_SKIP_SIGNATURE_VALIDATION", "0");
+
+        @Test
+        public void s3TestWithPresignedURL() throws IOException {
```
Comment (eddumelendez) [inline modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java:467]:
```suggestion
        public void presignedUrlShouldBeReadable() throws IOException {
```

File: modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java [hunk 4]
```diff
@@ -451,4 +455,53 @@ private String runAwsCliAgainstDockerNetworkContainer(String command) throws Exc
             return logs;
         }
     }
+
+    public static class WithEnvironmentVariables {
```
Comment (eddumelendez) [inline modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java:459]:
```suggestion
    public static class S3SkipSignatureValidation {
```

**Comment by eddumelendez** [comment]
@fokion can you take a look at the other two suggestions, please? I can not commit them for some reason in the GH UI :/

File: modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java (hunk 1 above)
Comment (fokion) [reply modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java:463]: Is that correct ? I thought we have to set the S3 service

Comment (eddumelendez) [reply modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java:463]: No since LocalStack 0.13 IIRC

Comment (fokion) [reply modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java:463]: ok removing then now

**Comment by eddumelendez** [comment]
Can you run `./gradlew spotlessApply`, please?

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks for your contribution, @fokion !

**Pull request merged**